crates/factor-key-value-sqlite/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "spin-factor-key-value-sqlite"
```

Comment:
nit: I'm fine with the crate name (even if it is a bit long), but I'm not sure I like that the directory inside the `crates` dir starts with `factor` even though there is not factor impl in it. I guess eventually this will just merge with `spin-key-value-sqlite` and we can use that name, right?

Reply:
Shouldn't we keep the `spin-` prefix to allow for eventual publishing on crates.io?